### DISCUSSION TITLE
Double time/half time for the metronome in muted mod?

### DISCUSSION THREAD

**Discussion by bytenekos** [ppy/osu#29928] [Ideas]
I think it'd be nice to be able to make the metronome go at double the speed/half the speed because (for example) some songs are at 120bpm but have 240bpm-like jumps/mapping and like-wise there could be maps where some people might prefer the opposite

**Comment by Tenexxt** [comment]
So basically an option to adjust the ticks that the metronome makes a sound on? 
Sounds nice tbh

**Reply by bytenekos** [reply]
Basically yeah. I tried playing [this](https://osu.ppy.sh/beatmapsets/426542#osu/920988) with the muted mod and felt like the metronome should be double the speed with this kind of mapping. Having the metronome at 127bpm with this kind of fast mapping feels too slow and "off rhythm" if that makes sense

**Comment by bdach** [comment]
Having it double time (and even quadruple time) was [tried before and it sounded like ass](https://github.com/ppy/osu/pull/9854#issuecomment-720683411). So I dunno if we wanna try that again.

**Reply by Tenexxt** [reply]
I don't know what's going on in that thread, but it could be resolved with just adding an option for making the metronome go off on 1/2, 1/4 etc. ticks and not only 1/1 ticks, since that's what the OP wants and just worded it weirdly (unless that was the approach taken there, then sorry for not understanding what you meant)

**Reply by bytenekos** [reply]
To clarify I guess what I was thinking of:
Either have the metronome play double/half the bpm (so in a 120bpm song, 2 bars would play in the time of 1 bar with double time. Half time would be the opposite)
Or have it skip/add ticks and keep the accents in sync with the original bpm (so in a 120bpm song, double time would play 1 accent and 7 ticks instead of 1 accent and 3 ticks, half time would play 1 accent and 1 tick (forgive me if I'm getting the terminology wrong))

Wrote the above because I have no idea how it sounded in that pr

By default it'd just be normal, but with an option in the customization menu to switch
